inputfiles/overridingTypes.jsonc

```diff
@@ -286,6 +286,138 @@
                         }
                     }
                 }
+            },
+            "ARIAMixin": {
+                "properties": {
+                    "property": {
+                        "ariaColCount": {
+                            "overrideType": "`${bigint}` | \"\""
```

Comment:
Does bigint have any difference than number when used here? Quickly tried but number doesn't reject huge number either.

Reply:
`${bigint}` only allows integers, `${number}` also allows floats.
So I used `${bigint}` for integer types and `${number}` for number types.

Also I allowed empty string `""` on these. Thinking people might use empty string instead of null.
But now since I'm also removing `string & {}` from enums, making them stricter. Should I also remove the empty string `""`?